orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/support/TargetServerGroup.groovy

```diff
@@ -58,6 +58,9 @@ class TargetServerGroup {
     } else if (loc.type == Location.Type.ZONE) {
       op.zone = loc.value
       op.zones = [loc.value]
+    } else if (loc.type == Location.Type.NAMESPACE) {
+      op.namespace = loc.value
```

Comment:
I'd also like to clean up the whole `location` thing. Have `location` be a field for every clouddriver operation, so we don't have to transform it constantly.